Create ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md (#2354)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md

---
date: ""

> title: QR code for TOTP is now shown correctly 

Reviewer: ```suggestion
title: QR code for two-factor authentication provided during TOTP setup now shows up correctly 
```

> title: otpAuth from chl or data (#6869) [GRAFT][release/cd] (#6883)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Support otpAuth from CHL or data
```

Reviewer 2: ```suggestion
title: QR code for TOTP is now shown correctly 
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60355
version: 1020.16.1
---

> otpAuth from chl or data (#6869) [GRAFT][release/cd] (#6883)

Author: The Claude AI suggested to use the following body:
```suggestion
To enhance the flexibility of the authentication process, support for the otpAuth parameter has been added. This parameter can now be provided either via the CHL interface or directly through the request data. With this change, users have more options to include the otpAuth value when authenticating, depending on their specific setup and requirements. Existing authentication flows will continue to work as before, while integrations that rely on passing otpAuth via CHL or data can now leverage this capability.
```

Reviewer 2: QR code for TOTP two-factor authentication was not showing correctly. Now the fix has been made to show QR code correctly for TOTP

Reviewer 2: ```suggestion
QR code for TOTP two-factor authentication was not showing correctly. Now the fix has been made to show QR code correctly for TOTP
```

> The QR code for the TOTP two-factor authentication was not displayed correctly. This issue has been fixed and now the QR code for TOTP shows up correctly.

Reviewer: ```suggestion
The QR code for the two-factor authentication provided during the TOTP setup was not displayed correctly. This issue has been fixed. The QR code is now properly visible and users can scan it with their authentication application.
```
